CustomDropDown/View/DropDownDisplayView.swift

```diff
@@ -42,19 +55,20 @@ public class DropDownDisplayView: UIView {
     // MARK: - Functions
     
     private func setupTitle(with tag: Int) {
-        title.tag = tag
+//        title.tag = tag
```

Comment:
Please remove the code you've commented yourself, if that won't be used. Don't remove the old commented code though. Thanks